configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql

```diff
@@ -0,0 +1,75 @@
+DELIMITER //
```

Comment:
@Michaelndula please maintain this as a create table statement without the stored precedure